install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -71,6 +71,11 @@ node with `*Key=KubernetesCluster,Value=clusterid*`.
 |xref:../../install_config/configuring_openstack.adoc#install-config-configuring-openstack[Configuring for OpenStack]
 |
 
+|Trident from NetApp
+|`netapp.io/trident`
+|xref:../../install_config/configuring_trident.adoc[Configuring for Trident]
```

Comment:
Looks like this content does not yet exist, so I will remove the link in the follow-up PR until we have something to link to.